### PR TITLE
`CHANGELOG.md` update and version bump `v0.2.2` -> `v1.0.0`

### PR DESCRIPTION
**Pull request by HalidOdat**
After this gets merged will publish the new version :)

### CODE REVIEW

**Review by jedel1043 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,60 +1,80 @@
...
 
-Internal improvements:
+### Breaking changes
+
+- Minimum rust version has been bumped from `1.36.0` to `1.64.0`.
+
+### Feature Enhancements
+
+- [FEATURE](https://github.com/boa-dev/ryu-js/pull/35):
+  ECMAScript specification complaint `Number.prototype.toFixed()` implementation. (@HalidOdat)
```
Comment (Razican) [inline CHANGELOG.md]: Should be "compliant"

File: src/lib.rs [hunk 2]
```diff
@@ -53,7 +53,7 @@
 //! notation.
 
 #![no_std]
-#![doc(html_root_url = "https://docs.rs/ryu-js/0.2.2")]
+#![doc(html_root_url = "https://docs.rs/ryu-js/1.0.0")]
```
Comment (Razican) [inline src/lib.rs]: Maybe we should just point to the latest version here, without pinning

**Review by Razican — APPROVED** [review]
Looking good! I found a typo and added a couple of comments, but you can merge once solved :)

File: src/lib.rs (hunk 2 above)
Comment (HalidOdat) [reply src/lib.rs]: I think it's better to remove it altogether. See https://github.com/rust-lang/api-guidelines/pull/230

EDIT: Removed it :)

**Pull request merged**